boards/BMS/Inc/App/App_TractiveSystem.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "App_SharedBinaryStatus.h"
```

Comment:
i dont think his header is needed any longer?

Reply:
deleted